_glossary/index.md

```diff
@@ -1,6 +1,8 @@
 ---
 layout: glossary
 title: Glossary
+
+languages: [zh-cn]
```

Comment:
I'm not sure about this setting -- I'm not sure what's the convention for scala-lang.org, but it seems it's possible to make the data available for the whole site by using _yaml data files_.

Reply:
The is supposed to be the correct way. See [cheatsheet](https://docs.scala-lang.org/cheatsheets/index.html) is doing the same thing in its [source code](https://github.com/scala/docs.scala-lang/edit/master/_overviews/cheatsheets/index.md).